tools/genesis-snapshot/options.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"github.com/iotaledger/goshimmer/packages/core/snapshotcreator"
+	"github.com/iotaledger/goshimmer/packages/protocol"
+	"github.com/iotaledger/goshimmer/packages/protocol/ledger/vm/devnetvm"
+	"github.com/iotaledger/hive.go/runtime/options"
+)
+
+var BaseOptions = []options.Option[snapshotcreator.Options]{
+	snapshotcreator.WithDatabaseVersion(protocol.DatabaseVersion),
+	snapshotcreator.WithVM(new(devnetvm.VM)),
+	// 7R1itJx5hVuo9w9hjg5cwKFmek4HMSoBDgJZN8hKGxih
+	snapshotcreator.WithGenesisSeed([]byte{
+		95, 76, 224, 164, 168, 80, 141, 174, 133, 77, 153, 100, 4, 202, 113, 104,
+		71, 130, 88, 200, 46, 56, 243, 121, 216, 236, 70, 146, 234, 158, 206, 230,
+	}),
+	snapshotcreator.WithGenesisTokenAmount(1000000000000000),
+	snapshotcreator.WithFilePath("snapshot.bin"),
+}
+
+var FeatureNetwork = []options.Option[snapshotcreator.Options]{
+	snapshotcreator.WithGenesisTokenAmount(1000000000000000),
+	snapshotcreator.WithFilePath("feature.snapshot"),
+	snapshotcreator.WithPeersPublicKeysBase58([]string{
+		"AZKt9NEbNb9TAk5SqVTfj3ANoBzrWLjR5YKxa2BCyi8X", // entry node (not present in docker network version)
```

Comment:
Entry node should be removed from here, because it doesn't even process blocks neither in Docker nor CI version.